tests/cmd/main_test.py

```diff
@@ -6,6 +6,7 @@
 import os.path
 
 import mock
+from six import StringIO
```

Comment:
Yeah, fair point—I retract my request to remove `six`. If we're using `six` here, though, we should probably use it elsewhere also—but I can take care of that after this is merged.